recipes/proj/all/patches/0001-Include-libdl-for-sqlite.patch

```diff
@@ -0,0 +1,24 @@
+From c0bf574d9f2464994cd7bec62cd36556a2b81d7b Mon Sep 17 00:00:00 2001
+From: Brennan Ashton <bashton@brennanashton.com>
+Date: Wed, 29 Jan 2020 22:59:46 -0800
+Subject: [PATCH] Include libdl for sqlite
+
+---
+ src/lib_proj.cmake | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/src/lib_proj.cmake b/src/lib_proj.cmake
+index bc27bafe..4e1025f1 100644
+--- a/src/lib_proj.cmake
++++ b/src/lib_proj.cmake
+@@ -431,6 +431,7 @@ set(PROJ_LIBRARIES ${PROJ_CORE_TARGET})
+ # hack, required for test/unit
+ set(PROJ_LIBRARIES ${PROJ_LIBRARIES} PARENT_SCOPE)
+ if(UNIX)
++  target_link_libraries(${PROJ_CORE_TARGET} -ldl)
```

Comment:
You can link to`${CMAKE_DL_LIBS}` [docs](https://cmake.org/cmake/help/latest/variable/CMAKE_DL_LIBS.html)

Reply:
Used `CONAN_PKG::sqlite3` instead as you recommended later